examples/simple_pendulum/run.py

```diff
@@ -39,3 +40,21 @@
 l = LagrangesMethod(L, [theta], forcelist=fl, frame=N)
 
 pprint(l.form_lagranges_equations())
+
+# Create a system from the lagranges equations
+sys = System(l)
+
+# Set up the system for simulation
+sys.times = linspace(0, 10, num=100)
+sys.constants = {m: 10, l: 5, g: 9.8}
+sys.initial_conditions = {theta: 60, thetad: 0}
+
+# Simulate the system
+out = sys.integrate()
+
+# Display the kinetic energy change in time (obtained from the particle in the
+# bodies list)
+KE = sys.body_kinetic_energies()
```

Comment:
Ok, that could work. You need to then have a way for the user to know which body/particle that the rows/columns of the array correspond to. And the kinetic energy of the whole system is important too. Would the user just sum along the axes of the numpy array for the total kinetic energy or would you precalculate that an it would be an extra row/col? This is where it would be good to show the desired output.